pgml-cms/blog/our-migration-from-aws-to-gcp-with-minimal-downtime.md

```diff
@@ -0,0 +1,134 @@
+---
+description: >-
+  Lessons learned from moving terabytes of real time data between cloud providers.
+featured: false
+tags: [engineering]
+---
+
+# Our migration from AWS to GCP with minimal downtime
```

Comment:
I would go with "Migrating databases from AWS to GCP with minimal downtime" to appeal to a more targeted audience.